Makefile

```diff
@@ -94,11 +95,11 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build -t ${IMG} .
+	docker buildx build --tag ${IMG} --platform ${image_arch_list} .
```

Comment:
not everyone has buildx installed. maybe we need to have different makefile commands to build and push images with buildx

Reply:
done